randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -2856,6 +2861,8 @@ Extra - asset_id: collision_camera_034
   * Extra - actor_def: actordef:actors/logic/breakabletilegroup/charclasses/breakabletilegroup.bmsad
   * Extra - actor_layer: breakables
   * Extra - tile_types: ('POWERBOMB',)
+  > Tunnel to Central Unit Access (Middle)
+      Trivial
```

Comment:
add a trivial connection to Upper-Right Alcove

Reply:
Check